index.html

```diff
@@ -176,26 +284,52 @@ <h3>Skills</h3>
         <li>Terminal</li>
       </ul>
     </div>
+        <div class="Aboutme--desktop">
+          <object class="img-dd_abtme" data="./desktopsvg/image_dots_with_teal.svg" type="image/svg+xml" ></object>
+        </div>
+
   </div>
   </section>
   </main>
 
-  <section class="Contactme">
 
+  <section class="Contactme">
       <div class="contactme-h"><a href="#" ><object data="./svgfiles/image_1.svg" type="image/svg+xml" ></object></a></div>
-      <div class="contactme--heading">
-        <h2>Get Started</h2>
-        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
-           Repellendus nesciunt, quos accusamus soluta unde
-            placeat quidem a maiores earum deleniti dolorem optio.</p>
-     
-      <form action="https://formspree.io/f/xwkjoqwr"  method="POST">
-        <input type="text" name="name" maxlength="30"  placeholder="Your name">
-        <input type="email" name="email" *required="" placeholder="Your email">
-        <textarea name="message" maxlength="500" placeholder="Enter your message here ..."></textarea>
-      <button type="submit">Start collaboration</button>
-    </form>
-  </div>
+    
+      <div class="contactme--desktop-img1">
+        <object data="./desktopsvg/image-16.svg" type="image/svg+xml" ></object>
+      </div>
+          <div class="contactme--heading">
+            <h2>Get Started</h2>
+            <p class="mobile--para">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Repellendus nesciunt, quos accusamus soluta unde
+                placeat quidem a maiores earum deleniti dolorem optio.</p>
+                <p class="desktop--para">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                   eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim 
+                   veniam, quis nostrud 
+                  exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+        
+              <form  class="mobile--form" action="https://formspree.io/f/xwkjoqwr"  method="POST">
+                <input type="text" name="name" maxlength="30"  placeholder="Your name">
+                <input type="email" name="email" *required="" placeholder="Your email">
```

Comment:
- [ ] please kindly add a background color #F9F9FA; to your contact section so it matches the figma design
yours
![Screenshot (835)](https://user-images.githubusercontent.com/65298048/221127019-8cbd20b1-8b0a-4ef7-89fb-cc93ed2e8e62.png)
figma
![Screenshot (836)_LI](https://user-images.githubusercontent.com/65298048/221127165-ded72a3f-12e4-4781-ac26-a04af858151b.jpg)